src/components/domain/ItemCard/ItemCard.tsx

```diff
@@ -0,0 +1,159 @@
+import styled from 'styled-components';
+import { Button } from '~components/base';
+
+interface requestInfoProp {
+  id: number;
+  title: string;
+  client: string;
+  due: string;
+  count: number;
+  amount: number;
+  method: string[];
+  material: string[];
+  status: string;
+}
+
+interface ItemCardProp {
+  requestInfo: requestInfoProp;
+}
```

Comment:
```suggestion
import { ApiReturnType } from '~types/index';

interface ItemCardProp {
  requestInfo: ApiReturnType;
}
```
1. Interface 이름은 대문자로 시작해야 하고
2. [types](https://github.com/OnBoarding-Park-is-best/week2-partners-dashboard/blob/develop/src/types/api.ts#L7)에 같은 type을 지정해둔 게 있어서 이걸 가져다 쓰시면 될 것 같습니다! 이건 제가 만들어 쓰고 안내를 못 드린 부분이라 ㅠㅠ 죄송해요!!